tox.ini

```diff
@@ -0,0 +1,144 @@
+# Differences from the starbase tox.ini:
```

Comment:
Do you know if the code coverage drop is related?

I know that git actions (rebase, push, merge) can confuse the code coverage.

Reply:
It's because I forgot to check the legacy tests. Those need coverage here, but they're being included as integration tests where we don't usually run coverage. Fixed!